docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: Hardware Key Support (YubiKey)
+description: Hardware Key Support
+---
+
+## Introduction
+
+By default, `tsh`, Teleport Connect, and other Teleport clients store a user's login session directly on their filesystem. If a user's filesystem is compromised, any of their active Teleport login sessions would also be compromised.
+
+[Per-session MFA](per-session-mfa.mdx) provides a way to require an MFA presence check with a user's login session, but this check is only used for starting sessions with Teleport services (SSH, Kube, DB, etc.). This means that non-session actions, like administrative commands run with `tctl`, can still be run with an exfiltrated login session. 
+
+To prevent these types of exfiltration attacks altogether, Teleport clients support hardware-based private keys. Unlike disk-based private keys, hardware-based private keys are generated and stored directly on a hardware device and are impossible to export. With this features, a user's login session is only functional with access to the user's hardware device.
+
+Additionally, this feature can be configured to require touch for every Teleport request, including non-session requests. With touch required, Hardware Key Support provides better security in comparison to our per-session MFA offering.
+
+<Admonition type="note" title="touch caching">
+
+  A user's touch is cached on the YubiKey for 15 seconds to prevent excessive touch prompts.
```

Comment:
This is the first time you mention YubiKeys specifically. Is the caching only supported for YubiKeys, or does this apply to all supported hardware keys?

Reply:
Currently, we only really support YubiKeys. there aren't any other (significant) PIV-compatible smart cards out there. See this section of the RFD covering this - https://github.com/gravitational/teleport/blob/cf595d8b0feeea863669d9e258186396ef225cb6/rfd/0080-piv-private-key.md#library

The touch cache is one of the things that may not be supported in other smart card implementations in the future.

I'll move this block below the PIV support block below, which is meant to clear up the change of wording from "hardware key" to "YubiKey".